### PR TITLE
Test regular grid

### PR DESCRIPTION
**Pull request by maxm17**
Could add more tests for half_cell_transmissibility.

### CODE REVIEW

File: tests/unit_tests/grid/test_regular_grid.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+def test_volume(basic_regular_grid):
+    # Act
+    volume = basic_regular_grid.volume()
+
+    # Assert
+    np.testing.assert_array_almost_equal(volume, 100000.0)
+
+
+def test_volume_faulted_grid(faulted_grid):
```
Comment (andy-beer) [inline tests/unit_tests/grid/test_regular_grid.py:46]: The faulted_grid is not a RegularGrid, so this test, along with test_thickness_faulted_grid() should probably be relocated to a different test module.

File: tests/unit_tests/grid/test_regular_grid.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+def test_thickness(basic_regular_grid):
+    # Act
+    thickness = basic_regular_grid.thickness()
+
+    # Assert
+    np.testing.assert_array_almost_equal(thickness, 20.0)
+
+
+def test_thickness_faulted_grid(faulted_grid):
```
Comment (andy-beer) [inline tests/unit_tests/grid/test_regular_grid.py:90]: See comment for test_volume_faulted_grid()

**Review by andy-beer — CHANGES_REQUESTED** [review]
Please move the two faulted_grid test functions into a different test module (the faulted grid is not a regular grid).

**Comment by maxm17** [comment]
@andy-beer Apologies, moved now

**Pull request merged**